Python/PRP/Animation/pyLineFollowMod.cpp

```diff
@@ -0,0 +1,145 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pyLineFollowMod.h"
+
+#include "pyAnimPath.h"
+#include <PRP/Animation/plLineFollowMod.h>
+#include "PRP/pyCreatable.h"
+#include "PRP/KeyedObject/pyKey.h"
+#include "PRP/Modifier/pyModifier.h"
+
+/* pyLineFollowMod */
+
+PY_PLASMA_NEW(LineFollowMod, plLineFollowMod)
+
+PY_METHOD_VA(LineFollowMod, addStereizer, "Params: stereizer") {
+    pyKey* stereizer;
+    if (!PyArg_ParseTuple(args, "O", &stereizer) || !pyKey_Check((PyObject*)stereizer)) {
+        PyErr_SetString(PyExc_TypeError, "addStereizer expects a plKey");
+        return NULL;
+    }
+    self->fThis->addStereizer(*stereizer->fThis);
+    Py_RETURN_NONE;
+}
+
+PY_METHOD_VA(LineFollowMod, delStereizer, "Params: idx") {
+    Py_ssize_t idx;
+    if (!PyArg_ParseTuple(args, "n", &idx)) {
+        PyErr_SetString(PyExc_TypeError, "delStereizer expects an int");
+        return NULL;
+    }
+    if (idx < self->fThis->getStereizers().size()) {
```

Comment:
I looked at the other cases where this check is done--they all cast idx to `size_t` and don't check for negative values. Is that acceptable or should I change my type in `PyArg_ParseTuple`?